### PR TITLE
GOK-394 | Fix Feedback received for Inward API

### PR DESCRIPTION
**Pull request by yenugukeerthana**
## Requirements

- [ ] This PR has a proper title that briefly describes the work done
- [ ] I have squashed / amended the comments to make it more redable
- [ ] I have included link to all the JIRA ticket('s)
- [ ] I wrote the code as a pair or atleast performed a self-review of my own code
- [ ] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/README.md) (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/inventory/aushada/aushada.jsx [hunk 1]
```diff
@@ -243,7 +244,23 @@ const Aushada = (props) => {
...
+    }
+  }, [selectedDate]);
+
+  const notificationMessage = enableEaushadhaInwardApi
+  ? 'No data is received for given inward date in this location. Could you please retry?'
+  : 'No data is received for the outward number. Could you please retry?';
+
+  const notificationAction = enableEaushadhaInwardApi
+    ? setStockEmptyResonseMessage
+    : setStockEmptyResonseMessage;
```
Comment (riyaTw) [inline src/inventory/aushada/aushada.jsx]: why both if and else condition of ternary is same here ?

File: src/inventory/aushada/aushada.jsx [hunk 2]
```diff
@@ -311,38 +328,33 @@ const Aushada = (props) => {
...
-              {ResponseNotification(
-                'info',
-                'info',
-                'No data is received for the outward number. Could you please retry?',
-                setStockEmptyResonseMessage,
-              )}
-            </div>
-          )}
+            {stockEmptyResonseMessage && (
+              <div style={{ paddingTop: '20px' }}>
```
Comment (riyaTw) [inline src/inventory/aushada/aushada.jsx]: Can we use 1rem here as well and extract out styles to css file

**Review by riyaTw — CHANGES_REQUESTED** [review]

File: src/inventory/aushada/aushada.jsx (hunk 1 above)
Comment (yenugukeerthana) [reply src/inventory/aushada/aushada.jsx]: We have updated the code and removed ternary condition.

File: src/inventory/aushada/aushada.jsx (hunk 2 above)
Comment (yenugukeerthana) [reply src/inventory/aushada/aushada.jsx]: Done

**Review by riyaTw — APPROVED** [review]

**Pull request merged**